src/test/java/brave/http/ITZipkinHttpClientImpl.java

```diff
@@ -15,7 +15,7 @@
 import ratpack.zipkin.internal.RatpackCurrentTraceContext;
 import ratpack.zipkin.internal.ZipkinHttpClientImpl;
 
-public class ITTracingFeature_Client extends ITHttpClient<HttpClient> {
+public class ITZipkinHttpClientImpl extends ITHttpAsyncClient<HttpClient> {
```

Comment:
@hyleung PS the base test classes here now have leak detection. So if you still have a span in scope when you go to close a span, a leak might have happened. This helped me notice we were never using the streaming variant in an integration test.